src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -604,11 +672,16 @@ def _populate_trtllm_output_json(
         text_input_headers: List[str],
         add_model_name: bool,
         add_stream: bool,
+        extra_inputs: Dict,
+        output_tokens_mean: int,
+        output_tokens_stddev: int,
         model_name: str = "",
-        extra_inputs: Dict = {},
     ) -> Dict:
         pa_json = LlmInputs._create_empty_trtllm_pa_json()
-        include_max_tokens = "max_tokens" not in extra_inputs
+        include_max_tokens = (
```

Comment:
is this tracking to include or not include?
variable name implies if its true, include max tokens
but the checks are looking for the absence of variable? am i just too tired to understand this